safe-global/safe-wallet-web
# fix: Hydration error on the NFTs page

## What it solves

- Resolves hydration error on the NFT Table when there is no linkTemplate (like on Sepolia)
- Navigates directly to the NFT page in the e2e test
- Resets balances to default token list for balances e2e test

## How to test it

1. Open the NFT page of a safe with NFTs on Sepolia
2. Observe no hydration error

## Checklist
